languages/csharp/exercises/concept/arrays/.docs/hints.md

```diff
@@ -8,28 +8,32 @@
 - As this method does _not_ depend on the current week's count, it is defined as a [`static` method][static-members].
 - There are [several ways to define an array][single-dimensional-arrays].
 
-### 2. Check how many birds visited yesterday
+### 2. Check how many birds visited today
 
 - Remember that the counts are ordered by day from oldest to most recent, with the last element representing today.
-- Accessing the second last element can be done either by using its (fixed) index (remember to start counting from zero) or by calculating its index using the [array's size][array-length].
+- Accessing the last element can be done either by using its (fixed) index (remember to start counting from zero) or by calculating its index using the [array's size][array-length].
 
-### 3. Calculate the total number of visiting birds
+### 3. Increment today's count
+
+- Set the element representing today's count to today's count plus 1.
+
+### 4. Check if there was a day with no visiting birds
+
+- The `Array` class has a [built-in method][array-indexof] to check at which index an element can be found. A special value is returned if a matching element could not be found.
```

Comment:
"A special value" is always -1 right? In that case I think this would be more explicit as:
"This method returns the first index where the element is found, -1 if there are no matching elements."

Reply:
I've rephrased!